### PR TITLE
Remove Idemixgen from Fabric produced binaries

### LINKED ISSUE #3575: Idemixgen was removed from the set of Fabric produced binaries but is still mentioned in ReadTheDocs
Idemixgen was removed from the set of Fabric produced binaries but is still mentioned in ReadTheDocs.

Need to scrub references of Idemixgen in Fabric ReadTheDocs since it is no longer provided in the set of Fabric binaries. Need to reference the new location at https://github.com/IBM/idemix/tree/main/tools/idemixgen.

Also mention in release notes for v2.4.x (Dave Enyeart can do this for you!)

### PR DESCRIPTION
**Pull request by Sayalikukkar**
#### Type of change

- Documentation update

#### Description
Removed Idemixgen from the documentation to reflect its removal from the set of Fabric produced binaries. Also created the seperate PR for idemix repo.

#### Related issues

Closes https://github.com/hyperledger/fabric/issues/3575

### CODE REVIEW

**Comment by denyeart** [comment]
Thank you for the update.

There is also a link to this document in a table of contents that needs to be removed at the same time - https://raw.githubusercontent.com/hyperledger/fabric/main/docs/source/ops_guide.rst

Also https://github.com/hyperledger/fabric/blob/main/docs/source/idemix.rst mentions that Fabric provides this tool. I think we'd want to update that wording from "Fabric provides two possible Idemix issuers" to "There are two possible Idemix issuers". And make "idemix tool" a hyperlink over to https://github.com/IBM/idemix.

Finally, I think we'd want this doc content to appear at https://github.com/IBM/idemix where the tool now resides, before removing it from Fabric docs, do you want to submit the PR to https://github.com/IBM/idemix as well to complete the move?

**Comment by Sayalikukkar** [comment]
@denyeart , Sure, I'm interested in helping with that. I can make the necessary updates to the Fabric documentation as you've described and also submit a pull request to the IBM idemix repository to ensure the content is properly moved.

File: docs/source/ops_guide.rst [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-   msp
-   certs_management.md
-   peer_ledger_snapshot
-   hsm.md
-   configtx
-   endorsement-policies
-   pluggable_endorsement_and_validation
-   access_control.md
-   idemix
-   idemixgen
```
Comment (denyeart) [inline docs/source/ops_guide.rst:16]: I meant to simply remove the idemixgen entry in this file, not the entire file!

**Comment by denyeart** [comment]
Thank you, looks good to me now.
But let's wait for @ale-linux to merge https://github.com/IBM/idemix/pull/41 to ensure we have the doc somewhere.

**Comment by Sayalikukkar** [comment]
Thank you for your feedback and guidance!

File: docs/source/idemix.rst [hunk 2]
```diff
@@ -56,10 +56,10 @@ Fabric components correspond to the user, issuer, and verifier in Idemix.
 * The Fabric Java SDK is the API for the **user**. In the future, other Fabric
   SDKs will also support Idemix.
 
-* Fabric provides two possible Idemix **issuers**:
+* There are two possible Idemix **issuers**:
 
    a) Fabric CA for production environments or development, and
-   b) the :doc:`idemixgen <idemixgen>` tool for development environments.
+   b) the `idemix <https://github.com/IBM/idemix>`_ tool for development environments.
```
Comment (denyeart) [inline docs/source/idemix.rst]:
```suggestion
   b) the `idemixgen <https://github.com/IBM/idemix/blob/main/tools/idemixgen/idemixgen.rst>`_ tool for development environments.
```

Ok, the idemixgen tool doc was merged into the idemix repository, so let's point directly to that now.

**Review by denyeart — APPROVED** [review]
Thank you for the quick responses, looks good now!

**Comment by denyeart** [comment]
Sorry, there is now a conflict in the doc, I think in the ops_guide table of contents. Could you resolve the conflict by rebasing locally and then re-push?

Also, it would be nice to squash the 3 commits into a single commit, see guidance at https://hyperledger-fabric.readthedocs.io/en/latest/github/github.html#updating-a-pull-request.

**Review by denyeart — APPROVED** [review]

**Comment by denyeart** [comment]
Perfect, thank you!

**Pull request merged**